jmix-gradle-plugin/src/main/groovy/io/jmix/gradle/EnhancingAction.groovy

```diff
@@ -71,6 +76,47 @@ class EnhancingAction implements Action<Task> {
         }
     }
 
+    /**
+     * Completes {@code classesInfo} with "empty" data stores in order to `persistence.xml` and `orm.xml` descriptors will be generated for them.
+     * "Empty" store is a store without entities. Such stores will not be added to {@code classesInfo} by
+     * the {@link io.jmix.gradle.EnhancingAction#collectClasses} discovery method.
```

Comment:
```suggestion
     * Completes {@code classesInfo} with "empty" data stores so that `persistence.xml` and `orm.xml` descriptors will be generated for them.
     * An "empty" store is a store without entities. Such stores will not be added to {@code classesInfo} by
     * the {@link io.jmix.gradle.EnhancingAction#collectClasses} discovery method.
```